app/styles/app/modules/request/preview.scss

```diff
@@ -0,0 +1,22 @@
+.request-configs-preview {
+  display: flex;
+
+  .left,
+  .right {
+    flex-grow: 1;
+    align-items: stretch;
+    width: 50%;
```

Comment:
For flex grid it should probably be `flex-basis`, shouldn't it?

Reply:
hmm, i'm not sure. i've copied these from a generator site. i'll have a look.

Reply:
turns out i could just remove the whole attribute: https://github.com/travis-ci/travis-web/pull/2399/commits/a9b2499df0d6c03693471fa1335407d28ca5f0de :)

Reply:
@svenfuchs it now seems to me that you can remove the entire rule (`.left, .right`), as stretching child items to equal widths is the default behavior for flex. So, basically, if you just remove the entire block, the appearance should not change at all.